.changeset/itchy-avocados-peel.md

```diff
@@ -0,0 +1,8 @@
+---
+'skuba': minor
+---
+
+lint: Manage .npmrc files for `pnpm` installations of skuba
+
+skuba now manages `.npmrc` files when using `pnpm` for compatibility purposes.
+skuba will attempt to ignore authTokens, and remove any `.gitignore` entries that ignore `.npmrc`.
```

Comment:
Nitpick: IIRC Changesets doesn't respect the Markdown spec for line breaks 😞

```suggestion
lint: Manage `.npmrc` for pnpm projects

skuba now manages `.npmrc` when a project uses `pnpm` to enable [dependency hoisting](https://pnpm.io/npmrc#dependency-hoisting-settings). It will attempt to strip sensitive auth tokens from the file, and remove `.gitignore` entries that ignore `.npmrc`.
```